core/include/thread.h

```diff
@@ -11,6 +11,97 @@
  * @ingroup     core
  * @brief       Support for multi-threading
  *
+ * Priorities
+ * ==========
```

Comment:
I have the feeling this section should have some link to the `sched.h` header in here somewhere ;-)